cmd/lotus-shed/main.go

```diff
@@ -118,7 +120,20 @@ func main() {
 		},
 	}
 
-	if err := app.Run(os.Args); err != nil {
+	// terminate early on ctrl+c
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	ctx, cancel := context.WithCancel(context.Background())
+	go func() {
+		<-c
+		cancel()
+		fmt.Println("Received interrupt, stopping... Press CTRL+C again to force stop")
```

Comment:
nit: this message confuses me a bit because it makes me think of SIGSTOP and that we are doing something like that.  Changing to "shutting down", and "force shut donw" would remove that confusion.